angr/analyses/ddg.py

```diff
@@ -415,12 +424,11 @@ def __getitem__(self, key):
             return DDGViewItem(self._ddg, pv, simplified=self._simplified)
 
     @property
-    def definitions(self):
+    def definitions(self) -> list[DDGViewItem]:
```

Comment:
Can you switch this to the List from typing? We're trying to keep compat with python3.8.

Reply:
Hi! I fixed it, thanks!